src/main/java/redis/clients/jedis/BuilderFactory.java

```diff
@@ -524,6 +524,55 @@ public String toString() {
 
   };
 
+  /**
+   * Create an Access Control Log Entry
+   * Result of ACL LOG command
+   */
+  public static final Builder<List<AccessControlLogEntry>> ACCESS_CONTROL_LOG_ENTRY_LIST = new Builder<List<AccessControlLogEntry>>() {
+
+    Map<String,Builder> mappingFunctions = createDecoderMap();
```

Comment:
```java
private static final Map<String,Builder> mappingFunctions = createDecoderMap(); 

private static Map<String, Builder> createDecoderMap() { 
```

Reply:
* done for `private` & `final`
* cannot use `static` (static inner class)